### ISSUE TITLE
DNX Core

### ISSUE THREAD

**Issue by acastaner** [BigGranu/MusicBrainz#4]
I was wondering if you were planing to port this API to DNX Core 5.0?

**Comment by BigGranu** [comment]
I'm not sure. Where is the need?

**Comment by acastaner** [comment]
The need is that, apparently, you can't use a library (such as this one) in a DNX Core priject if it's not compatible with DNX Core too :/

**Comment by BigGranu** [comment]
I'm sorry. I do not know how can I create a DNX Core Library.
Will you help me?

**Comment by acastaner** [comment]
Absolutely, in fact, I have already started :) I'll fork the repo and show you what I can come up with.

**Comment by acastaner** [comment]
I've made good progress, I have only one remainder issue in Search.cs. You use MethodBase.GetCurrentMethod(), but Microsoft removed this method from their new API so I'm not sure how to fix it.

**Comment by acastaner** [comment]
I have fixed the issue, you should now be able to check my changes. Note that I added a new project to the solution, in theory this should be split in two (or merged into one - but that would require more work). I'm about to test the API too, I haven't run tests yet but now it does compile.

It's available here: https://github.com/acastaner/MusicBrainz

Note you'll need VS Studio 2015 or VS Code + DNVM to check this.

**Comment by BigGranu** [comment]
Good job. :+1:  I'll see it tomorrow precisely. Thank you first.

**Comment by acastaner** [comment]
I've pushed a new code which now makes this compatible with DNX 4.5.1 (not only DNX Core 5.0) and replaced the legacy API WebResponse with the new HttpWebResponse recommended to use.

**Comment by acastaner** [comment]
Would you be okay if I forked this project and publish a separate NuGet package (ie: MusicBrainz.DNX) ? I would of course give you credit for the original work.

**Comment by BigGranu** [comment]
I have take a look in the code. I Download it and Compiled, but i have not tested. 
But I think it's OK. :+1:  So you can fork and publish it. MusicBrainz.DNX is a good Name.
Thanks.

**Comment by acastaner** [comment]
Great, thanks! I've put it there with a MIT license:

https://github.com/ribandelle/MusicBrainz.DNX

I'll push to NuGet sometime later.
